docs/content/developers/design-keuzes.md

```diff
@@ -214,9 +214,7 @@ in het geval van validatiefouten:
 
 Op het [forum](https://forum.pdok.nl/t/formaat-foutafhandeling-input-validatie-api-50/1848)
 is nagevraagd hoe er moet omgegaan worden met meerdere fouten op hetzelfde veld,
-met de conclusie dat dit beter gespecifieerd moet worden.
-
-In ZDS 2.0 kiezen we ervoor om elke fout op een veld als apart object op te
+met de conclusie dat dit beter gespecifieerd moet wordenIn ZDS 2.0 kiezen we ervoor om elke fout op een veld als apart object op te
```

Comment:
Hier is wat fout gegaan - de punt en newlines om nieuwe alinea te starten zijn verdwenen.

Reply:
Zie onderstaande opmerking